lab02/sections/results.tex

```diff
@@ -30,21 +32,37 @@ \section{Results}\label{sec:results}
 			12.0 & 0.136 & -17.303 & -33   \\
 		\bottomrule
 	\end{tabular}
-	\caption{Response of Fig. \ref{fig:schematic} to frequency change ($V_x = \SI{1}{\volt}$)}
+	\caption{Response of Fig.~\ref{fig:schematic} to frequency change ($V_x = \SI{1}{\volt}$)}
 	\label{table:data}
 \end{table}
 
+\subsection{Analysis}
+
+Table~\ref{table:data} shows the response of the phase lag circuit to a range of frequencies, starting at 500\si{\hertz} then 1\si{\kilo\hertz} steps from 1\si{\kilo\hertz} to 12\si{\kilo\hertz}.
```

Comment:
`\SI{number}{units}` will prevent numbers and their units from being split by a linebreak.